molns.py

```diff
@@ -1454,6 +1454,29 @@ def run(self, args, config_dir=None):
             Command('import',{'filename.json':None},
                 function=MOLNSProvider.provider_import),
         ]),
+                
+        SubCommand('service',[
```

Comment:
This was not supposed to be there… it was an experiment I started on, when I was thinking about how to generalize the compute backend to different types (like Spark, Celery etc.)

-------------------------------------------
Andreas Hellander
Associate Professor, Scientific Computing
Division of Scientific Computing
Department of Information Technology
Uppsala University

On Sep 17, 2015, at 12:36 AM, Brian Drawert <notifications@github.com<mailto:notifications@github.com>> wrote:


In molns.py<https://github.com/Molns/molns/pull/45#discussion_r39694142>:

> @@ -1454,6 +1454,29 @@ def run(self, args, config_dir=None):
>              Command('import',{'filename.json':None},
>                  function=MOLNSProvider.provider_import),
>          ]),
> +
> +        SubCommand('service',[


We should discuss this. This would make 'service' in the molns vocabulary as a clone/alias of 'provider'. What is the motivation to do that?

—
Reply to this email directly or view it on GitHub<https://github.com/Molns/molns/pull/45/files#r39694142>.